service/frontend/dcRedirectionHandler.go

```diff
@@ -46,18 +44,15 @@ type (
 
 	// DCRedirectionHandlerImpl is simple wrapper over frontend service, doing redirection based on policy
 	DCRedirectionHandlerImpl struct {
+		resource.Resource
+
 		currentClusterName string
-		timeSource         clock.TimeSource
-		domainCache        cache.DomainCache
-		metricsClient      metrics.Client
 		config             *Config
 		redirectionPolicy  DCRedirectionPolicy
 		tokenSerializer    common.TaskTokenSerializer
-		service            service.Service
 		frontendHandler    workflowserviceserver.Interface
-		clientBeanProvider clientBeanProvider
 
-		startFn func() error
+		startFn func()
 		stopFn  func()
```

Comment:
i guess, these 2 functions can be removed

Reply:
I think we have these 2 functions because wfHandler is stored as type `workflowserviceserver.Interface` in dcRedirectionHandler and that interface doesn't have `Start()` and `Stop()` method.